packaging/setup/plugins/ovirt-engine-setup/ovirt-engine-grafana-dwh/core/config.py

```diff
@@ -232,7 +232,10 @@ def _customization_sso(self):
         ),
         condition=lambda self: (
             self.environment[ogdwhcons.CoreEnv.ENABLE] and
-            self.environment[ogdwhcons.ConfigEnv.NEW_DATABASE]
+            (
+                self.environment[ogdwhcons.ConfigEnv.NEW_DATABASE] or
+                self.environment[ogdwhcons.KeycloakEnv.KEYCLOAK_ENABLED]
+            )
```

Comment:
Hmm, wouldn't above condition cause, that keycloak for grafana would be reconfigured on each engine-setup execution? If so, can't we detect somehow if the configuration already exists?

Reply:
It's done on keycloak side where   ogdwhcons.KeycloakEnv.KEYCLOAK_ENABLED =  okkcons.Coreenv.Enable AND okkcons.DBEvn.NEW_DATABASE.   
That actually makes want to rename grafana KEYCLOAK_ENABLED  to something like KEYCLOAK_CONFIGURATION_REQUIRED